cardano_node_tests/tests/test_plutus_spend_build.py

```diff
@@ -1562,6 +1562,133 @@ def test_guessing_game(
 
         assert "The Plutus script evaluation failed" in str(excinfo.value)
 
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    def test_two_scripts_spending_one_fail(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addrs: List[clusterlib.AddressRecord],
+    ):
+        """Test locking two Tx outputs with two different Plutus scripts in single Tx, one fails.
+
+        Uses `cardano-cli transaction build` command for building the transactions.
+
+        * create a Tx output with a datum hash at the script addresses
+        * try to spend the locked UTxOs
+        * check that the expected error was raised
+        """
+        temp_template = common.get_test_id(cluster)
+        amount = 50_000_000
+
+        script_fund = 500_000_000
+        collateral_fund = 750_000_000
+
+        plutus_op1 = plutus_common.PlutusOp(
+            script_file=plutus_common.ALWAYS_SUCCEEDS_PLUTUS,
+            datum_file=plutus_common.DATUM_42_TYPED,
+            redeemer_cbor_file=plutus_common.REDEEMER_42_CBOR,
+        )
+        plutus_op2 = plutus_common.PlutusOp(
+            script_file=plutus_common.ALWAYS_FAILS_PLUTUS,
+            datum_file=plutus_common.DATUM_42_TYPED,
+            redeemer_cbor_file=plutus_common.REDEEMER_42_CBOR,
+        )
+
+        # Step 1: fund the Plutus scripts
+
+        script_address1 = cluster.gen_payment_addr(
+            addr_name=f"{temp_template}_addr1", payment_script_file=plutus_op1.script_file
+        )
+
```

Comment:
This space is inconsistent with [line 1607](https://github.com/input-output-hk/cardano-node-tests/pull/1032/files#diff-bd5de8ea3de581961243504343003963907839f1f7c5c418a6371f8ff263920cR1607) :-)